src/acl.c

```diff
@@ -121,9 +121,8 @@ typedef struct {
     uint64_t allowed_commands[USER_COMMAND_BITS_COUNT/64];
     /* allowed_firstargs is used by ACL rules to block access to a command unless a
      * specific argv[1] is given (or argv[2] in case it is applied on a sub-command).
-     * For example, a user can use the rule "-select +select|0" to block all
-     * SELECT commands, except "SELECT 0".
-     * And for a sub-command: "+config -config|set +config|set|loglevel"
+     * WARNING: Allowing a first arg of an otherwise blocked command is a misuse of ACL 
+     * and may get disabled in the future.
```

Comment:
i'm not certain the warning is needed here.
this information is already present in both redis.conf, and also in the comment above ACLSetSelector.
we should either copy the same text from there, or better yet, trim some excessive info from, but not add more.

i see two valid options:
1. trim just the outdated line, the one that mentions `loglevel`, but keep the `select` example which is still valid.
2. trim the `select` example lines too, but IMHO no need to add any additional repeated info.

Reply:
I prefer approach 2